contrib/containerlab/bgpv2/multi-homing/Makefile

```diff
@@ -1,14 +1,13 @@
 # version of cilium to deploy
-HELM_CHART ?= oci://quay.io/cilium-charts-dev/cilium
-VERSION ?= 1.16.0-dev-dev.247-main-8c97a2119d
+VERSION ?= v1.16.0-rc.0
```

Comment:
How about setting Renovate here? So that we don't need to bump Cilium version manually.

Reply:
I'll work on it. Need to explore how to set renovate. Do you have any example/guide on it ? 
I am following this one : https://docs.cilium.io/en/latest/contributing/development/renovate/

Reply:
We need to add a regexManager entry that matches this `VERSION ?= <cilium version>` I think. The tricky part is how we should maintain this rule per branch. For the main branch, we'll update to the latest version of the currently developed version, but for the stable branches, we only want to update the patch version (e.g. for v1.16 branch, we don't want to upgrade it to v1.17 even it is released).

https://github.com/cilium/cilium/blob/main/.github/renovate.json5#L636-L727

Reply:
As discussed offline, I will follow this up in separate PR.